app/plugins/Wallet/js/buttons.js

```diff
@@ -106,7 +106,7 @@ addResultListener('coin-sent', function(result) {
 // Lock or unlock the wallet
 eID('lock-pod').onclick = function() {
 	var state = eID('lock-status').innerHTML;
-	if (!wallet.unlocked && state === 'Unencrypted') {
+	if (!wallet.encrypted && state === 'Create Wallet') {
 		encrypt();
```

Comment:
The wallet is not encrypted and the state is 'create wallet', then call encrypt()? That seems like a weird way to trigger the encryption dialog.